_action_files/word2post.sh

```diff
@@ -8,21 +8,21 @@ fi
 
```

Comment:
I really like the changes you made to this file.  Can we revert the changes to the other files and keep only the changes to this file?